app/panel/components/InsightsPromoModalContent.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Insights Promo Modal Component
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import React from 'react';
+
+const InsightsPromoModalContent = (props) => {
+	const {
+		handleGoAwayClick,
+		handleSignInClick,
+		handleTryInsightsClick,
+		XButton,
+	} = props;
+	return (
+		<div className="InsightsModal__content flex-container flex-dir-column align-middle">
+			{XButton}
```

Comment:
I'm still confused as to why this outer `<div />` and `<ExitButton />` can't be moved out of each individual modal content component and into the parent `<PromoModal />` component, as you were saying. From what I can tell, the only difference between them is the main class name (which you can determine in `<PromoModal />` with the `classnames` library), and the `<ExitButton />` click handler (which you're already determining in `<PromoModal />`, but then pass down to the child components).

Reply:
Moved the ExitButton out! Making a fragment inside each `PromoModalContent` class messed up the CSS, but the Exit button is inside the PromoModal.jsx component nonetheless!